### ISSUE TITLE
properties with `type: date-only` should reject `datetime` values

### ISSUE THREAD

**Issue by jstoiko** [raml-org/raml-js-parser-2#490]
```
#%RAML 1.0
title: People API

/people:
  post:
    body:
      application/json:
        properties:
          birthday:
            type: date-only
```
`{birthday: '1999-12-31T21:00:00'}` is accepted, it shouldn't be

After trying different combinations, it seems like any characters after yyyy-mm-dd are accepted, e.g. `{birthday: '1999-12-311111111111111111111111'}`

**Comment by ddenisenko** [comment]
Reproduced, thank you for reporting.

**Comment by KonstantinSviridov** [comment]
to be released in 1.1.5
